UT4MasterServer/Controllers/PersonaController.cs

```diff
@@ -75,8 +75,8 @@ public async Task<IActionResult> GetAllAccounts()
 		foreach (var account in accounts.Take(1000))
 		{
 			var obj = new JObject();
-			obj.Add("id", account.ID.ToString());
-			obj.Add("displayName", account.Username);
+			obj.Add("ID", account.ID.ToString());
+			obj.Add("Username", account.Username);

```

Comment:
![image](https://user-images.githubusercontent.com/905878/215381169-465a7de3-236b-4e75-b0da-fd7d3b7076ad.png)

Reply:
I'm not sure what you're trying to say. The endpoints in the account controller are used by the game, and thus must use that json format. The endpoints in persona controller were created solely for the front end and are returning the Account model as is. We did this because we didn't want to modify the json format the game uses, but the front end needed access to the flags/roles

Reply:
If it is for FE only, we're good

Reply:
Can you add info about it to `<summary>` tag for this controller?